### DISCUSSION TITLE
Copilot Chat does not recognize open file in Visual Studio

### DISCUSSION THREAD

**Discussion by inseroaj** [community/community#78653] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

In Visual Studio 17.8.3 (not Code), GitHub Copilot chat version 0.1.1982.7773
Start a new project, open a file, open Copilot chat window, and ask it to "/explain".
It just replies: "I'm sorry, but without the specific code you want me to explain, I can't provide a detailed explanation. Please provide the code you want me to explain"

I have to provide the filename by using #file, or use the inline chat instead.

If I understand correctly, it should be able to detect the currently open file(s) ?

**Comment by rheapatel23** [comment]
Hi @inseroaj, there are two entry points for Copilot Chat, the Chat window and Inline chat (accessed for alt / in a file). 
Currently, the Chat window doesn't assume the file you are referring to, so using the "#" to reference a file will ensure better results versus Inline which has context on the file you are in.  

We are digging into what customers want as assumed knowledge by our chat, and would love to dig in deeper to understand more about you experience. Please provide your email and I will follow up directly. 

Thanks, 
Rhea 
Product Manager for GitHub Copilot for Visual Studio

**Comment by inseroaj** [comment]
So you're basically saying it's working as intended.
I expected the chat window to be aware of the open file, due to this section in the documentation:
https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide#asking-github-copilot-chat-questions-about-your-code-1

Maybe it refers to the inline chat, however this section immediately follows a section describing the chat window 
https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide#asking-your-first-question-1

That's why I expected it to work that way.

**Reply by rheapatel23** [reply, marked as answer]
Yes it is working as expected. Thanks for this feedback! I would like to dig into your experience with this more - could you provide an email and our team will follow up :)

**Reply by inseroaj** [reply]
I think the question has been fully answered and explained, so I don't see the need for exchanging email addresses at this point :)
